src/pages/index.tsx

```diff
@@ -1,41 +1,371 @@
+  ],
+  imports: [
+    BrowserModule,
+    UppyAngularDashboardModule,
+  ],
+  providers: [],
+  bootstrap: [AppComponent],
+})
+class {}
+`;
+
+const svelteCode = `
+<main> 
+  <Dashboard 
+      uppy={uppy} 
+      plugins={["Webcam"]}
+  />
+</main>
+
+<script>
+import { Dashboard } from '@uppy/svelte'
+
+import Uppy from '@uppy/core'
+import Webcam from '@uppy/webcam'
+
+const uppy = new Uppy().use(Webcam);
+</script>
+`;
+
+const providersIcons = [
+	'box.svg',
+	'unsplash.svg',
+	'googledrive.svg',
+	'dropbox.svg',
+	'instagram.svg',
+	'facebook.svg',
+	'onedrive.svg',
+];
+
+const frameworks = [
+	{ name: 'React', Icon: IconReact, code: reactCode },
+	{ name: 'Vue', Icon: IconVue, code: vueCode },
+	{ name: 'Svelte', Icon: IconSvelte, code: svelteCode },
+	{ name: 'Angular', Icon: IconAngular, code: angularCode },
+];
 
 export default function Home(): JSX.Element {
-	const { siteConfig } = useDocusaurusContext();
+	const [framework, setFramework] = useState(frameworks[0].name);
+
 	return (
-		<Layout
-			title={`Hello from ${siteConfig.title}`}
-			description="Description will go into a meta tag in <head />"
-		>
-			<HomepageHeader />
-			<main>
-				<HomepageFeatures />
+		<Layout description="Description will go into a meta tag in <head />">
+			<header className={styles.header}>
+				<h1>Sleek, modular open source JavaScript file uploader</h1>
+
+				<p>
+					Uppy fetches files locally and from remote places like Dropbox or
+					Instagram. With its seamless integration, reliability and ease of use,
+					Uppy is truly your best friend in file uploading.
+				</p>
+
+				<Link className={styles.button} to="/docs/quick-start">
+					Get started
+				</Link>
+
+				<figure className={styles.quote}>
+					<blockquote cite="https://stackshare.io/posts/top-developer-tools-2017">
+						<p>“Top 10 tools of the year”</p>
+					</blockquote>
+					<figcaption>
+						<a
+							href="https://stackshare.io/posts/top-developer-tools-2017"
+							rel="noopener"
+							target="_blank"
+						>
+							Stackshare
+						</a>
+					</figcaption>
+				</figure>
+
+				<figure className={styles.quote}>
+					<blockquote cite="https://books.producthunt.com/bestof2017">
+						<p>“The best product launches”</p>
+					</blockquote>
+					<figcaption>
+						<a
+							href="https://books.producthunt.com/bestof2017"
+							rel="noopener"
+							target="_blank"
+						>
+							Product Hunt
+						</a>
+					</figcaption>
+				</figure>
+
+				<figure className={styles.quote}>
+					<blockquote cite="https://twitter.com/smashingmag/status/1097870169043546112">
+						<p>“Soooo useful”</p>
+					</blockquote>
+					<figcaption>
+						<a
+							href="https://twitter.com/smashingmag/status/1097870169043546112"
+							rel="noopener"
+							target="_blank"
+						>
+							Smashing Magazine
+						</a>
+					</figcaption>
+				</figure>
+			</header>
+
+			<main className={styles.main}>
+				<section className={styles['section-dashboard']}>
```

Comment:
When the user's the screen real estate allows it, I propose to increase the width of this particular section — to fit the desktop version of Dashboard on larger screens. And we might also slightly increase the width of the code example block thanks to that.

Reply:
:+1: I'd propose to make it full-width and code below or hidden in a tab. I think the centered full-width live example is more important than code as a homepage hero thingy.